src/plugins/intel_cpu/tests/functional/single_layer_tests/split.cpp

```diff
@@ -54,7 +54,7 @@ class SplitLayerCPUTest : public testing::WithParamInterface<splitCPUTestParams>
         int axis;
         ElementType netPrecision;
         InputShape inputShapes;
-        InferenceEngine::SizeVector outIndices;
+        ov::Shape outIndices;
```

Comment:
Here the variable "outIndices" is a shape or a index? If it is index, I think this change will make some misunderstand.

Reply:
`InferenceEngine::SizeVector` is `std::vector<size_t>` and `ov::Shape` also is `std::vector<size_t>`, they don't have too much difference.  Even `outIndices` is indices, it is a vector of indices, `ov::Shape` can cover it.

Reply:
Maybe 'std::vector<size_t>' will be better here to avoid misunderstanding.

Reply:
Anyway, `ov::Shape` is used here in a wrong context, please, change to `std::vector` in next PR

Reply:
ok, there is other similar place in line 28, I changed that line but forgot this line, will change it, thanks!